### PR TITLE
Error page and menu update

### PR DESCRIPTION
**Pull request by whitneykidd**
Error pages updated for 404, 422, 500
- In development, you'll have to visit `/404.html` in order to view

Hamburger (mobile) navbar now functions correctly!
- mobile menu button has been updated to match theme

Desktop menu
- Now has transparent grey background for readability
- Has logo in left corner that routes to '/' (I just added a logo to know that it's possible, totally down to change it)

User messages
- Now appear in the navbar in the space between the logo and 'Take the Fear Quiz'

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -10,8 +10,9 @@
...
-//= require rails-ujs
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
 //= require activestorage
 //= require turbolinks
-//= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+//= require_tree .
```
Comment (whitneykidd) [inline app/assets/javascripts/application.js:18]: //= require_tree . **always** has to come last in application.js

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -10,8 +10,9 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require rails-ujs
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
```
Comment (whitneykidd) [inline app/assets/javascripts/application.js:15]: We weren't requiring these before, which is why the hamburger nav wasn't working

File: app/assets/stylesheets/style.scss [hunk 3]
```diff
@@ -9987,8 +9987,8 @@ body {
 #mainNav .navbar-toggler {
   font-size: 80%;
   padding: 0.75rem;
-  color: #64a19d;
-  border: 1px solid #64a19d;
+  color: #ee9d1c;
```
Comment (whitneykidd) [inline app/assets/stylesheets/style.scss:9990]: Color now matches the yellow/mustard on the page

File: app/assets/stylesheets/style.scss [hunk 4]
```diff
@@ -10287,15 +10288,10 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
...
-  color:#e9c318;
-  border: 1px solid;
-  border-color:black;
-  font-family: "Arial";
+  color:#EE9D1C;
   font-weight: bold;
-  font-style: italic;
-  padding-left: 40px;
 }
+
```
Comment (whitneykidd) [inline app/assets/stylesheets/style.scss:10294]: Notice now appears in the nav and doesn't require all this fancy ish

File: public/404.html [hunk 5]
```diff
@@ -1,67 +1,76 @@
...
-    <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
-    </div>
-    <p>If you are the application owner check the logs for more information.</p>
-  </div>
+  <div class="error">404</div>
+  <br /><br />
+  <span class="info">Spooks not found</span>
+
```
Comment (whitneykidd) [inline public/404.html:74]: This was a fun practice dive into animation 👀

File: public/422.html [hunk 6]
```diff
@@ -1,67 +1,106 @@
...
+      overflow: default;
+    }
+
+    .info {
+      text-align: center;
+      font-family: 'Gilda Display', serif;
+      font-size: 15px;
+      font-style: italic;
+      text-align: center;
+      width: 400px;
```
Comment (whitneykidd) [inline public/422.html:43]: The only real difference between these error pages is the .info{width} as the messages are longer

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (melatran) [reply app/assets/javascripts/application.js:18]: good to know

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -10,14 +10,16 @@
...
-    <section class=<%= type %>>
-      <p class="user-message"><%= message %></p>
-    </section>
-  <% end %>
-  <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
-    <div class="container">
-        <a class="navbar-brand js-scroll-trigger" href="/">I Scream Arcade</a>
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
+      <a class="navbar-brand js-scroll-trigger" href="/">
+        <img class="icon" src="https://cdn2.iconfinder.com/data/icons/funeral-filloutline/64/scythe-reaper-cultures-spooky-terror-scary-fear-horror-death-halloween-256.png" alt="zombie" border="0"></a>
```
Comment (melatran) [inline app/views/layouts/application.html.erb:15]: This image name `spooky-terror-scary-fear-horror-death` is just great. And fits the theme. I'm down for this logo

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (perryr16) [reply app/assets/javascripts/application.js:18]: weird. `//` feels like a comment

File: app/assets/stylesheets/style.scss (hunk 3 above)
Comment (perryr16) [reply app/assets/stylesheets/style.scss:9990]: better than bisque

File: app/assets/stylesheets/style.scss (hunk 4 above)
Comment (perryr16) [reply app/assets/stylesheets/style.scss:10294]: clean. nice

File: app/views/layouts/application.html.erb (hunk 7 above)
Comment (perryr16) [reply app/views/layouts/application.html.erb:15]: haha, yeah I like that. I enjoy the 64 bit nature

**Pull request merged**